py/tests/test_subscriptions.py

```diff
@@ -0,0 +1,15 @@
+from tests.test_case import *
+from app.pcasts.dao import subscriptions_dao # pylint: disable=C0413
+
+class SubscriptionsTestCase(TestCase):
+
+  def setUp(self):
+    super(SubscriptionsTestCase, self).setUp()
+    Subscription.query.delete() # delete all existing subscriptions
+    db_session_commit()
+
+  def test_get_subscriptions(self):
+    assert not subscriptions_dao.get_user_subscriptions(1)
+
+if __name__ == '__main__':
```

Comment:
is this in case we don't want to run the tests using `nose`?

Reply:
Oh I assumed `nose` needed this to run...but yeah I guess that works too lmao